src/Bridge/ExceptionListener.php

```diff
@@ -0,0 +1,74 @@
+<?php
+declare(strict_types=1);
+
+namespace Jaeger\Symfony\Bridge;
+
+use Jaeger\Span\SpanManagerInterface;
+use Jaeger\Tag\ErrorTag;
+use Jaeger\Tracer\TracerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\ExceptionEvent;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\Event\TerminateEvent;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+class ExceptionListener implements EventSubscriberInterface
+{
+    private $tracer;
+    private $spanManager;
+    private $globalSpanHandler;
+    private $exceptionExist;
+
+    public function __construct(
+        TracerInterface $tracer,
+        SpanManagerInterface $spanManager,
+        GlobalSpanHandler $globalSpanHandler
+    ) {
+        $this->tracer = $tracer;
+        $this->spanManager = $spanManager;
+        $this->globalSpanHandler = $globalSpanHandler;
+        $this->exceptionExist = false;
+    }
+
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            ExceptionEvent::class => ['onKernelException', 0],
+            RequestEvent::class => ['onRequest', 28],
+            TerminateEvent::class => ['onTerminate', 4097],
+        ];
+    }
+
+    public function onKernelException(ExceptionEvent $event): void
+    {
+        if (HttpKernelInterface::MASTER_REQUEST !== $event->getRequestType()) {
```

Comment:
Not completely sure what are you trying to achieve here.
Do you want to mark all requests, e.g. not only master requests with error? It makes more sense to mark ONLY the request that caused the exception with an error tag as parent requests could've recovered from the exception in som internal sub-requests and didn't produce the error on their own.
It will be easier to detect where the error actually happened in the trace treee itself, as there will be fewer number of spans with error

Reply:
Yeah, you're right.